indico/modules/events/timetable/blueprint.py

```diff
@@ -34,6 +36,10 @@
 _bp.add_url_rule('/manage/timetable/', 'management', RHManageTimetable)
 _bp.add_url_rule('/manage/timetable/', 'timetable_rest', RHTimetableREST, methods=('POST',))
 _bp.add_url_rule('/manage/timetable/<int:timetable_entry_id>', 'timetable_rest', RHTimetableREST, methods=('PATCH',))
+_bp.add_url_rule('/manage/timetable/<int:timetable_entry_id>/move', 'move_timetable_entry', RHLegacyTimetableEntryMove,
+                 methods=('POST', 'PATCH'))
+_bp.add_url_rule('/manage/timetable/<int:timetable_entry_id>/change-start-date', 'change_start_date',
```

Comment:
I remembered about changing stuff in `vars_js.py` but forgot to change the blueprint.. :-1: for me